include/odp/api/spec/traffic_mngr.h

```diff
@@ -108,6 +108,18 @@ extern "C" {
  * be set to. It is in units of 1000 bytes/second.
  */
 
+/**
+ * @def ODP_TM_MIN_SHAPER_BURST
```

Comment:
This makes the limit part of the ABI, which is problematic. I think these kind of things should be in the capabilities so that different implementations of the same ABI can have their own limits.

There are many simiilar defines with the same problem in the traffic manager API. It would be good to move them too to capabilities.